airbyte-integrations/connectors/destination-ragie/Dockerfile

```diff
@@ -0,0 +1,9 @@
+
+FROM airbyte/python-connector-base:1.1.0
```

Comment:
Remove this file and use the base image in metadata.yaml instead.

Reply:
done